core/domain/question_domain.py

```diff
@@ -39,12 +43,21 @@
 CMD_ADD_QUESTION_SKILL = 'add_question_skill'
 CMD_REMOVE_QUESTION_SKILL = 'remove_question_skill'
 
+CMD_CHANGE_ROLE = 'change_role'
```

Comment:
Is this used? How about the other constants below this one?

Please remove stuff that is not used -- they cause confusion otherwise.

Reply:
Yes removed.